docs/en/tools-and-utilities/static-files-disk-uploader.md

```diff
@@ -0,0 +1,82 @@
+---
+slug: /en/operations/utilities/static-files-disk-uploader
+title: clickhouse-static-files-disk-uploader
+keywords: [clickhouse-static-files-disk-uploader, utility, disk, uploader]
+---
+
+# clickhouse-static-files-disk-uploader
+
+Outputs a data directory containing metadata for a specified ClickHouse table. This metadata can be used to create a ClickHouse table on a different server containing a read-only dataset backed by a `web` disk.
+
+Do not use this tool to migrate data. Instead, use the [`BACKUP` and `RESTORE` commands](/docs/en/operations/backup).
+
+## Usage
+
+```
+$ clickhouse static-files-disk-uploader [args]
+```
+
+## Commands
+
+|Command|Description|
+|---|---|
+|`-h`, `--help`|Prints help information|
+|`--metadata-path [path]`|The path containing metadata for the specified table|
+|`--test-mode`|Enables `test` mode, which submits a PUT request to the given URL with the table metadata|
+|`--link`|Creates symlinks instead of copying files to the output directory|
+|`--url [url]`|Web server URL for `test` mode|
+|`--output-dir [dir]`|Directory to output files in `non-test` mode|
+
+## Retrieve metadata path for the specified table
+
+When using `clickhouse-static-files-disk-uploader`, you must obtain the metadata path for your desired table.
+
+1. Run the following query specifying your target table and database:
+
+<br />
+
+```sql
+SELECT data_paths
+  FROM system.tables
+  WHERE name = 'mytable' AND database = 'default';
+```
```

Comment:
This might help any padding/margin issues:

````suggestion
    ```sql
    SELECT data_paths
      FROM system.tables
      WHERE name = 'mytable' AND database = 'default';
    ```
````

Reply:
Does the padding look incorrect? It looks OK in my local environment.

Reply:
It might be fine, I didn't actually look at a preview. But this suggestion was to remove the need for the `<br>` tags.